cmd/autoowners/main.go

```diff
@@ -294,18 +326,25 @@ func pullOwners(directory string, whitelist []string) ([]string, error) {
 
 	config := filepath.Join(operatorRoot, "config")
 	templates := filepath.Join(operatorRoot, "templates")
-	for _, orgRepo := range orgRepos {
-		logrus.WithField("orgRepo", fmt.Sprintf("%+v", *orgRepo)).Info("handling repo ...")
+	for i, orgRepo := range orgRepos {
+		logrus.WithField("index", i).
+			WithField("orgRepo", fmt.Sprintf("%s/%s", orgRepo.Organization, orgRepo.Repository)).
```

Comment:
You're doing the `org/repo` Sprintf in multiple places, maybe it would be a nice `orgRepo` helper method

Reply:
Will do this when refactoring ...

Reply:
Done.